2018/ELMI-Dublin/OMERO-Workshop/index.html

```diff
@@ -0,0 +1,192 @@
+<!doctype html>
+
+<html lang="en">
+
+    <head>
+        <meta charset="utf-8">
+
+        <title></title>
+
+        <meta name="apple-mobile-web-app-capable" content="yes" />
+        <meta name="apple-mobile-web-app-status-bar-style" content="black-translucent" />
+
+        <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
+
+        <link rel="stylesheet" href="reveal-js/css/reveal.min.css">
+        <link rel="stylesheet" href="reveal-js/css/theme/default.css" id="theme">
+
+        <!-- OME presentation overrides of the default theme -->
+        <link rel="stylesheet" href="css/ome.css">
+
+        <!-- For syntax highlighting -->
+        <link rel="stylesheet" href="reveal-js/lib/css/zenburn.css">
+
+        <!-- If the query includes 'print-pdf', use the PDF print sheet -->
+        <script>
+            document.write( '<link rel="stylesheet" href="reveal-js/css/print/' + ( window.location.search.match( /print-pdf/gi ) ? 'pdf' : 'paper' ) + '.css" type="text/css" media="print">' );
+        </script>
+
+    </head>
+
+    <body class="ome_theme">
+
+        <div class="reveal">
+
+            <!-- Any section element inside of this container is displayed as a slide -->
+            <div class="slides">
+
+                <!-- title page -->
+                <section>
+                    <h1>OMERO workshop</h1>
+                    <p>
+                        ELMI <br>
+                        Dublin, June 2018 <br>
+                        Jean-Marie Burel, Petr Walczysko
+                    </p>
+                </section>
+                
+                <section>
+                    <h2>Outline</h2>
+                    <ul>
+                        <li>Intro: OME</li>
+                        <li>What is OMERO?</li>
+                        <li>Workshop: Using OMERO</li>
+                    </ul>
+                </section>
+
+
+                <section>
+                    <h2>A typical microscopy workflow</h2>
+
+                    <img src="https://downloads.openmicroscopy.org/presentations/2014/Technique-Tapas-OMERO-Figure/images/typical-workflow.png">
+                </section>
+
+                
+                <section>
+                    <h2>OME: Interoperability</h2>
+
+                    <img src="https://downloads.openmicroscopy.org/presentations/2014/Technique-Tapas-OMERO-Figure/images/ome-informatics.png">
+                </section>
+
+
+                <section>
+                    <h2>OME: What We Do</h2>
+
+                    <img src="images/ome-what-we-do.png">
+                </section>
+
+
+                <section>
+                    <h2>OMERO is a server with clients</h2>
+
+                    <img src="https://downloads.openmicroscopy.org/presentations/2014/technique_tapas_nov_5/images/omero-platform-simple.png">
```

Comment:
If these images exist in this repository, I would avoid linking to another presentation via downloads.oo.org. Maybe move them under `template/images`?

Reply:
This is a cut and paste from a previous presentation =>
We have few presentations to review to avoid that